contrib/go-pg/go-pg/pg_go.go

```diff
@@ -0,0 +1,54 @@
+package pggo
+
+import (
+	"context"
```

Comment:
Can you group these just like the rest of the project? Start with the standard lib as one group, followed by other imports and lastly the dd-trace-go group.